src/util/queryBankBalances.ts

```diff
@@ -0,0 +1,21 @@
+import { QueryClient, createProtobufRpcClient } from '@cosmjs/stargate';
+import { Tendermint34Client } from '@cosmjs/tendermint-rpc';
+import { QueryClientImpl } from 'cosmjs-types/cosmos/bank/v1beta1/query';
+import type { Coin } from '@cosmjs/stargate';
+import type { HttpEndpoint } from '@cosmjs/tendermint-rpc';
+
+export const queryBankBalances = async (
+  address: string,
+  rpc: HttpEndpoint,
+): Promise<Coin[]> => {
+  const tendermint = await Tendermint34Client.connect(rpc);
```

Comment:
ambient authority. hm.

> No modules should export powerful objects (for example, objects that close over fs or process.env). -- https://github.com/Agoric/agoric-sdk/wiki/OCap-Discipline

I suppose we have agreed on an exception to that for UI code? I'd like to see that in the README or the like.

Reply:
Noted, I don't know if I have a clear way to explain the philosophy as it pertains to UI code at the moment though.

Reply:
I think we have agreed agreed on an exception to that for UI code and I agree it would be valuable to have in the README of this repo to refer to. Not a blocker imo

Reply:
agreed, not a blocker.

meanwhile, I realize pointing to the wiki doesn't automatically make a link the other way, where citing an issue does: https://github.com/Agoric/agoric-sdk/issues/2160